fctesting/sandbox/Dockerfile

```diff
@@ -0,0 +1,58 @@
+ARG TMP_BUILD_DIR=/tmp/build
+
+FROM rust:1.32 as build
+
+ARG TMP_BUILD_DIR
+ARG FIRECRACKER_SRC_DIR="/firecracker"
+ARG FIRECRACKER_BUILD_DIR="$FIRECRACKER_SRC_DIR/build"
+ARG CARGO_REGISTRY_DIR="$FIRECRACKER_BUILD_DIR/cargo_registry"
+ARG FIRECRACKER_VERSION="0.15.0"
+
+ENV CARGO_HOME=/usr/local/rust
+ENV RUSTUP_HOME=/usr/local/rust
+ENV PATH="$CARGO_HOME/bin:$PATH"
+ENV CARGO_TARGET_DIR="$FIRECRACKER_BUILD_DIR/cargo_target"
+ENV FC_TEST_DATA_PATH="$TMP_BUILD_DIR"
+
+# Install system dependecies
+#
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends \
+        musl-tools
+
+# Install the Rust toolchain
+#
+# Install 1.32.0 due to Firecracker crashing with 148 code
+RUN mkdir "$TMP_BUILD_DIR" \
+    	&& rustup install 1.32.0 \
+	&& rustup override set 1.32.0 \
+        && rustup target add x86_64-unknown-linux-musl \
+    && cd /
+
+# Install Firecracker and set binaries to TMP_BUILD_DIR
+RUN git clone https://github.com/firecracker-microvm/firecracker.git
+WORKDIR firecracker
+RUN git checkout tags/v"$FIRECRACKER_VERSION"
+RUN cargo build --release --features vsock --target-dir "$TMP_BUILD_DIR"
+RUN cp "$TMP_BUILD_DIR"/x86_64-unknown-linux-musl/release/firecracker "$TMP_BUILD_DIR"
+RUN cp "$TMP_BUILD_DIR"/x86_64-unknown-linux-musl/release/jailer "$TMP_BUILD_DIR"
+
+FROM golang:1.12
+
+ARG TMP_BUILD_DIR
+ENV FC_TEST_DATA_PATH="$TMP_BUILD_DIR"
+ARG FIRECRACKER_GO_SDK_SRC_DIR="/firecracker-go-sdk"
+ADD . /firecracker-go-sdk/
+COPY --from=build "$TMP_BUILD_DIR" "$TMP_BUILD_DIR"
+
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends \
+        binutils-dev \
+        git \
+        make
```

Comment:
`golang:1.12` already has `git` and `make`. I suspect binutils are also no longer needed. Hence can we drop this entire `RUN`?